tests/algorithms/algorithm_settings.py

```diff
@@ -11,17 +11,31 @@
 from typing import Any, Dict, Optional, Type
 
 import pytest
-from torch.utils.data import Dataset
+from torch.utils.data import DataLoader
 
 import composer
 from composer import Algorithm
 from composer.algorithms import (AGC, EMA, SAM, SWA, Alibi, AugMix, BlurPool, ChannelsLast, ColOut, CutMix, CutOut,
-                                 Factorize, GhostBatchNorm, LabelSmoothing, LayerFreezing, MixUp, NoOpModel,
-                                 ProgressiveResizing, RandAugment, SelectiveBackprop, SeqLengthWarmup, SqueezeExcite,
-                                 StochasticDepth)
+                                 Factorize, FusedLayerNorm, GhostBatchNorm, LabelSmoothing, LayerFreezing, MixUp,
+                                 NoOpModel, ProgressiveResizing, RandAugment, SelectiveBackprop, SeqLengthWarmup,
+                                 SqueezeExcite, StochasticDepth)
 from composer.models import ComposerResNet
 from composer.models.base import ComposerModel
 from tests import common
+from tests.fixtures.synthetic_hf_state import (make_synthetic_bert_dataloader, make_synthetic_bert_model,
+                                               make_synthetic_gpt2_dataloader, make_synthetic_gpt2_model)
+
+simple_bert_settings = {
+    'model': make_synthetic_bert_model(),
```

Comment:
Instead of putting the constructed model in the dictionary, and `model` be just the function?
Then, `get_alg_model` will construct it.

Same thing re: dataloader and below for `simpel_gpt2_settings`.